web/models/user.ex

```diff
@@ -35,4 +35,14 @@ defmodule RemoteRetro.User do
     |> unique_constraint(:email)
     |> validate_format(:email, ~r/@/)
   end
+
```

Comment:
we  could/ should make it private though, until such time as its needed by other modules.